src/data/ha-state-history-data.js

```diff
@@ -210,13 +208,18 @@ class HaStateHistoryData extends LocalizeMixin(PolymerElement) {
     startTime,
     endTime,
     cacheConfig,
-    localize,
-    language
+    localize
   ) {
-    if (!this.hass) return;
-    if (cacheConfig && !cacheConfig.cacheKey) return;
-    if (!localize || !language) return;
-    this._madeFirstCall = true;
```

Comment:
Is this being set anywhere else? If not then does it make sense to use it in `hassChanged`